py/h2o_wave/conda/meta.yaml

```diff
@@ -15,20 +15,33 @@ about:
 
 requirements:
   build: 
+    {% if NO_ARCH == "0" %}
+    - python={{ python_version }}
+    {% endif %}
     {% for dep in pyproject['build-system']['requires'] %}
     - {{ dep.lower() }}
     {% endfor %}
   run:
+    {% if NO_ARCH == "0" %}
+    - python={{ python_version }}
+    {% endif %}
     {% for dep in project['dependencies'] %}
     - {{ dep.lower() }}
     {% endfor %}
 source:
   path: ..
 
 build:
-  noarch: python
   script: python -m pip install . -vv
+  string: py{{ python_version | replace(".", "") }}
   entry_points:
     {% for module, entrypoints in project['scripts'].items() %}
       - {{ module }} = {{ entrypoints }}
-    {% endfor %}
\ No newline at end of file
+    {% endfor %}
+  include_recipe: False
+  script_env:
+    - NO_ARCH
+  {% if NO_ARCH == "1" %}
+  noarch: python
+  string: "0"
```

Comment:
What is this `string` for?

Reply:
To override `string: py{{ python_version | replace(".", "") }}` in case of noarch builds.
However better would be doing something like this
```yaml
{% if NO_ARCH == "0" %}
string: py{{ python_version | replace(".", "") }}
{% endif %}
```
and remove `string: "0"` to let conda do its job with naming noarch builds.